no-computer.md

```diff
@@ -11,41 +11,70 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Imagine yourself lost in the streets of Mexico. You walk up to someone and
+say "Biblioteca" as it is the only Spanish word you know. The kind caballero
+tells you "Siga en esta calle para dos kilómetros, después a la derecha y
+camina por cuatro cuadras." You still have no idea where the library is.
+If only there was some universal way to communicate directions.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Well it's definitely a language as It's a way to communicate information
+between people. It's pretty domain specific as it is designed for communicating
+directions.
+
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+Over the summer, I took an American Sign Language class and was very intrigued
+with the nonverbal way to communicate.
+
 
 ### Domain
 _Describe the project's domain in five words._
 
+Visual communication to convey directions.
+
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The user would diagram the directions out in the signing space in front of
```

Comment:
I wonder how hard it would be to convey visual directions without pen and paper though since the signs could easily be misinterpreted.